ui/sections/storage/inspectors/disk.reel/disk.html

```diff
@@ -55,7 +55,8 @@
                 "controller": {"@": "owner"}
             },
             "bindings": {
-                "context": {"<-": "@owner.context"}
+                "context": {"<-": "@owner.context"},
+                "canDelete": {"<-": "!!@owner.object._canDelete"}
```

Comment:
You shouldn't use Delete button to do this, but add a custom button instead. See https://github.com/freenas/gui/blob/master/ui/sections/storage/inspectors/volume.reel/volume.html for an example (look for Scrub button).